.github/workflows/android-deploy.yaml

```diff
@@ -0,0 +1,87 @@
+name: Android Prod Deploy
```

Comment:
Did you use a template for this? Obviously if it's all you 👍 but if there's reference documentation it would be nice to see a quick comment!

Reply:
Mostly I just copied the Android build and upload parts of the existing CI workflow.